src/table.ts

```diff
@@ -1504,23 +1535,27 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
     );
     const mutationErrorsByEntryIndex = new Map();
 
-    const onBatchResponse = (
-      err: ServiceError | PartialFailureError | null
-    ) => {
-      // TODO: enable retries when the entire RPC fails
-      if (err) {
-        // The error happened before a request was even made, don't retry.
-        callback(err);
-        return;
+    const isRetryable = (err: ServiceError | null) => {
+      if (pendingEntryIndices.size === 0 || numRequestsMade > maxRetries) {
```

Comment:
should it numRequestsMade >= maxRetries?

Reply:
Yes, I think numRequestsMade > maxRetries is more correct, but currently it's implemented as `numRequestsMade > maxRetries`, so I'm wondering if I should change that?